### PR TITLE
fix: Use `import` in vanilla-ts fragment.

### PR DESCRIPTION
**Pull request by FabianLars**
### What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Docs
- [ ] New Binding issue #___
- [ ] Code style update
- [ ] Refactor
- [ ] Build-related changes
- [ ] Other, please describe:

### Does this PR introduce a breaking change?

- [ ] Yes, and the changes were approved in issue #___
- [x] No

### Checklist
- [ ] When resolving issues, they are referenced in the PR's title (e.g `fix: remove a typo, closes #___, #___`)
- [x] A change file is added if any packages will require a version bump due to this PR per [the instructions in the readme](https://github.com/tauri-apps/create-tauri-app/blob/dev/.changes/readme.md).
- [ ] I have added a convincing reason for adding this feature, if necessary

### CODE REVIEW

**Comment by FabianLars** [comment]
And that's why you test changes locally before you open a PR 💃

**Review by amrbashir — COMMENTED** [review]
What's the reason for this? This is a vanilla template so I think imports should be from the global tauri but I am fine with the change if there is a reasonable drive behind it.

**Comment by FabianLars** [comment]
Imo it just doesn't make sense to not use the npm package if vite (and typescript) is used, especially if we install the api package anyway.

**Comment by FabianLars** [comment]
Oh and the "trigger" for this PR was a help thread on discord where someone was confused why we use the global object in a typescript project https://discord.com/channels/616186924390023171/625037620996734986/1025553047436406835

**Comment by amrbashir** [comment]
Makes sense.

File: packages/cli/fragments/fragment-vanilla-ts/src/vite-env.d.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import type * as TauriApiTypes from "@tauri-apps/api";
+export {};
```
Comment (amrbashir) [inline packages/cli/fragments/fragment-vanilla-ts/src/vite-env.d.ts]: why not just remove the whole line?

**Review by amrbashir — CHANGES_REQUESTED** [review]
Can you also move the api package to normal dependencies section instead of dev?

File: packages/cli/fragments/fragment-vanilla-ts/src/vite-env.d.ts (hunk 1 above)
Comment (FabianLars) [reply packages/cli/fragments/fragment-vanilla-ts/src/vite-env.d.ts]: Without it you get this error:
> Augmentations for the global scope can only be directly nested in external modules or ambient module declarations.

And `export {}` was the only thing i found on the internet to get rid of it. [1](https://stackoverflow.com/a/59499895) / [2](https://bobbyhadz.com/blog/typescript-augmentations-for-global-scope-can-only-be-directly-nested)

Comment (amrbashir) [reply packages/cli/fragments/fragment-vanilla-ts/src/vite-env.d.ts]: right, just remove `declare global` and `export {};`. this should be enough
```ts
interface Window {
  greet(): void;
}
```

**Review by amrbashir — APPROVED** [review]

**Pull request merged**